<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сопровождение НТП</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="tables">
    <property name="title">
     <string> Данные</string>
    </property>
    <addaction name="nir"/>
    <addaction name="prog"/>
    <addaction name="vuz"/>
   </widget>
   <widget class="QMenu" name="analys">
    <property name="title">
     <string>Анализ данных</string>
    </property>
    <addaction name="analys1"/>
    <addaction name="analys2"/>
    <addaction name="analys3"/>
   </widget>
   <widget class="QMenu" name="fin">
    <property name="title">
     <string>Финансирование</string>
    </property>
    <addaction name="orderbtn"/>
   </widget>
   <widget class="QMenu" name="progactions">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="aboutbtn"/>
    <addaction name="helpbtn"/>
    <addaction name="closebtn"/>
   </widget>
   <addaction name="tables"/>
   <addaction name="analys"/>
   <addaction name="fin"/>
   <addaction name="progactions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="nir">
   <property name="text">
    <string>Таблица с НИР</string>
   </property>
  </action>
  <action name="prog">
   <property name="text">
    <string>Таблица с НТП</string>
   </property>
  </action>
  <action name="vuz">
   <property name="text">
    <string>Данные о вузах</string>
   </property>
  </action>
  <action name="closeaction">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="analys1">
   <property name="text">
    <string>Распределение НИР по вузам</string>
   </property>
  </action>
  <action name="analys2">
   <property name="text">
    <string>Распределение по НТП</string>
   </property>
  </action>
  <action name="analys3">
   <property name="text">
    <string>Характер НИР</string>
   </property>
  </action>
  <action name="orderbtn">
   <property name="text">
    <string>Выдать распоряжение</string>
   </property>
  </action>
  <action name="aboutbtn">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="helpbtn">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="closebtn">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
